src/components/cards/pane-volunteer/styles.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components'
+
+export const Card = styled.section`
+  max-width: 768px;
+  border-radius: ${({ theme }) => theme.borderRadius.base}px;
+  overflow: hidden;
+`
+
+export const Cover = styled.div<{ url: string }>`
+  height: 287px;
+  width: 100%;
+  background-image: url(${({ url }) => url});
```

Comment:
Chápu správně, že obrázek je zde jako `background-image` z důvodu sémantiky HTML a struktury stránky?

Reply:
Přiznám se, že jsem tam původně měla klasický `img` ale pak jsem narazila na [tenhle PR](https://github.com/cesko-digital/web/pull/89) a to řešení přes `background-image` mi přišlo elegantnější. Tak jsem se inspirovala 😄.

Přijde mi, že ten cover obrázek je spíš ilustrační, než že by nesl nějaký význam a bylo třeba doplňovat `alt` popisek. Ale možná se pletu. Jestli ti přijde vhodnější vložit tam klasický obrázek, stačí říct 😉.

Reply:
Já sám nevím, co je _ideální řešení_ v tomto případě. Z hlediska struktury webu bez stylů dává smysl mít obrázek spíše jako `background-image`. Na druhou stranu zde ztrácíme možnost lazy načtení. 

Vzhledem k tomu, že se jedná o detail, který můžeme kdykoliv později dořešit, tak jsem se ptal spíše pro zajímavost a byl bych OK s obojím. 🙂